metatron/src/main/java/uk/gov/ida/eidas/metatron/core/dto/KeyStoreModule.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.ida.eidas.metatron.core.dto;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JavaType;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.security.KeyStore;
+import java.security.KeyStoreException;
+import java.security.NoSuchAlgorithmException;
+import java.security.cert.CertificateException;
+import java.util.Base64;
+
+public class KeyStoreModule extends SimpleModule {
```

Comment:
I can't immediately tell what this is used for.

Reply:
It's a Jackson / ObjectMapper thing.  They call them modules.  So do I?